src/SDKs/NetApp/NetApp.Tests/ResourceTests/SnapshotTests.cs

```diff
@@ -143,8 +143,15 @@ public void PatchSnapshot()
                     Tags = dict
                 };
 
-                var resource = netAppMgmtClient.Snapshots.Update(snapshotPatch, ResourceUtils.resourceGroup, ResourceUtils.accountName1, ResourceUtils.poolName1, ResourceUtils.volumeName1, ResourceUtils.snapshotName1);
-                Assert.True(resource.Tags.ToString().Contains("Tag1") && resource.Tags.ToString().Contains("Value1"));
+                try
```

Comment:
nit: Consider using `Assert.Throws` here

Reply:
ok. If it's acceptable I will look at this for the next version which is gated GA in the next few weeks.